searchcore/src/vespa/searchcore/proton/server/documentdb.cpp

```diff
@@ -338,7 +338,7 @@ DocumentDB::enterOnlineState()
 {
     // Called by executor thread
     // Ensure that all replayed operations are committed to memory structures
-    _feedView.get()->forceCommit(_feedHandler->getSerialNum());
+    _feedView.get()->forceCommit(CommitParam(_feedHandler->getSerialNum(), true));
```

Comment:
Why are we using _forceUpdateStats=true_ here? That is not equal to the previous behaviour. The same comment applies for other changes in this file.